style.css

```diff
@@ -0,0 +1,558 @@
+  font-weight: 900;
+  color: #595959;
+  font-size: 1.25rem;
+  letter-spacing: -0.02rem;
+  padding-top: 2rem;
+}
+
+#locationP {
+  font-family: "Gothic A1", sans-serif;
+  font-size: 0.8rem;
+  color: #868686;
+  padding: 0 0 2rem 0;
+}
+
+#mainProgram {
+  width: 100vw;
+  background-image: url(./img/backgrounds/gray.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.subTitle {
+  padding-top: 1rem;
+  color: #efefef;
+  font-weight: 400;
+}
+
+#cardsContainer {
+  margin: 1rem;
+}
+
+.cardPicture {
+  max-height: 5rem;
+}
+
+.underline {
+  margin-top: 10px;
+  margin-bottom: 1.5rem;
+  width: 2rem;
+}
+
+.mainProgramCards {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: rgba(255, 255, 255, 0.13);
+  margin-bottom: 3px;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+}
+
+.cardIcon {
+  margin: 0.5rem;
+}
+
+.mainProgramCardsTitle {
+  color: #ff5438;
+  font-size: 0.9rem;
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.mainProgramCardsP {
+  font-size: 0.8rem;
+  width: 12rem;
+  color: #efefef;
+  line-height: 25px;
+}
+
+#joinButton {
+  width: 18rem;
+  height: 4rem;
+  text-decoration: none;
+  background-color: #ff4f38;
+  font-weight: 900;
+  color: #efefef;
+  font-size: 1rem;
+  border: 1px #ff5438;
+  margin: 2rem;
+}
+
+#featured {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.featuredSubTitle {
+  color: black;
+  font-size: 2rem;
+  margin-top: 2rem;
+}
+
+.gridContainer {
+  display: block;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 0;
+  grid-row-gap: 0;
+  margin-top: 2rem;
+  margin-bottom: 3rem;
+  margin-left: 1rem;
+  margin-right: 1rem;
+  gap: 2rem;
+}
+
+.div1 {
+  grid-area: 1 / 1 / 2 / 2;
+}
+
+.div2 {
+  grid-area: 2 / 1 / 3 / 2;
+}
+
+.div3 {
+  grid-area: 3 / 1 / 4 / 2;
+}
+
+.div4 {
+  grid-area: 1 / 2 / 2 / 3;
+}
+
+.div5 {
+  grid-area: 2 / 2 / 3 / 3;
+}
+
+.div6 {
+  grid-area: 3 / 2 / 4 / 3;
+}
+
+.cardTrans:hover {
+  transform: scale(1.1);
+  transition: 300ms ease-in-out;
+}
+
+.cardContainer {
+  display: flex;
+  gap: 15px;
+  margin-bottom: 1rem;
+}
+
+.cardTitle {
+  font-size: 1rem;
+}
+
+.cardUnderline {
+  translate: 0 -6px;
+  width: 1.2rem;
+  height: 1rem;
+}
+
+.cardsSubtitle {
+  color: #f78c89;
+  font-size: 0.78rem;
+  font-weight: 700;
+  line-height: 171%;
+}
+
+.cardsP {
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 171%;
+}
+
+#grayUpperMenu {
+  display: none;
+}
+
+.menuLinks {
+  display: none;
+}
+
+.a-menuLink:hover {
+  transform: scale(1.1);
+  color: #ec4327;
+}
+
+.a-menuLink:link {
+  text-decoration: none;
+}
+
+.a-menuLink:visited {
+  text-decoration: none;
+}
+
+#footer {
+  display: none;
+}
+
+#endTitle {
+  display: none;
+}
+
+#partner {
+  display: none;
+}
+
```

Comment:
- [OPTIONAL] Good job for removing the button 👏I think it would be better to add partner's and footer for home page  for mobile version like the way these sections are present in about page. The home page is looking a bit incomplete without these section 😓 Since the previous reviewer didn't asked about this, I'm suggesting it as optional. However, I highly recommend implementing this ⛏🤓

Reply:
Hi @KanzaTahreem
Thanks for your review. Can you please add a screenshot of what you suggested to change?